sig-k8s-infra/charter.md

```diff
@@ -1,129 +1,184 @@
-  preferably an approver.
+- We are responsible for enforcing policy on what is considered in-scope and
+  out-of-scope for project infrastructure (and thus, where we spend our
+  infrastructure budget)
+- Crafting such policy is done in collaboration with the Steering Committee
+  (owns project spending) and SIG Architecture (owns kubernetes definition)
+- We MAY delete or scope down infrastructure in the event of unexpected or
+  undue spend
 
-- **Medium risk** changes may impact existing contributor workflows, should be
-  easy to roll back, and may impact all of the project's repos. These should
-  be shared with the appropriate SIGs, and may require a [lazy consensus]
-  issue with [kubernetes-dev@] notice.
+e.g. SIG K8s Infra will deny requests to host artifacts for projects that are
+formerly part of or adjacent to the kubernetes project (e.g. helm, cri-o)
 
-- **High risk changes** likely break existing contributor workflows, may be
-  difficult to roll back, and likely impact all of the project's repos. These
-  require a consultation with SIG Contributor Experience as well as any other
-  owning SIGs, and a [lazy consensus] issue with [kubernetes-dev@] notice.
+##### Public Names
 
-### Out of Scope
+- We are responsible for enforcing policy on what is considered appropriate
+  or inappropriate for the names of public-facing entities such as DNS
+  records and Google Group names
+- Crafting such policy is done in collaboration with the Steering Committee,
+  SIG Architecture, and SIG Contributor Experience
 
-- We are not responsible for maintaining infrastructure which has not yet been
-  migrated to the CNCF. For example, problems with prow.k8s.io should be routed
-  to SIG Testing.
+e.g. Group names that are used to community upon behalf of the project are
+vetted by SIG Contributor Experience, group names that are used for RBAC or
+IAM bindings are vetted by SIG K8s Infra.
+
+##### Secrets and Credentials
+
+- We are responsible for ensuring secure storage and retrieval of secrets
+  such as passwords, tokens, keys, etc.
+- However, we are NOT responsible for ensuring the value of those secrets
+  is valid.
+- We MAY delete or deactivate secrets in the event of a security-related
+  incident
+
+e.g. SIG Contributor Experience is responsible for ensuring valid Slack API
+credentials exist for proper functioning of slack-infra
+
+##### Security Response
+
+- Overriding all of the above, we MAY revoke, delete, or deactivate
+  infrastructure, services or access in the event of a security-related
+  incident.
+- This depends on responsiveness of the owning SIG, and urgency and severity
+  of the incident being responded to
+
+e.g. SIG K8s Infra may force rotation of prow build cluster credentials if
+appropriately credentialed members of SIG Testing are not available
+
+### Out of scope
+
+Unless explicitly stated otherwise above, or list as a subproject of this SIG,
```

Comment:
This line reads strangely.

Reply:
What I'm trying to get at is
```suggestion
We are not responsible for code that runs _on_ project infrastructure, with the exception of:
- subprojects of this SIG (as listed in the-more-continuously-kept-up-to-date sigs.yaml)
- code we share responsibility for (as listed in the [Cross-cutting and Externally Facing Processes] section)
```

Reply:
The suggested change is clearer to me.

Reply:
Made something close to the suggested change: https://github.com/kubernetes/community/pull/5928/files#diff-b55cad1162db37049c36ff01b2892c020ecd9b8ce4a89284850d9bc2b6a6a25aR176-R182